tooling/did-method-registry-entry.yml

```diff
@@ -41,6 +41,18 @@ properties:
     description: https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry
     type: string
     maxLength: 512
+  implementation:
+    description: A URL to an implementation of the DID Method
+    type: string
+    maxLength: 512
+  conformanceReport:
+    description: A URL to a DID Test Suite Conformance Report for the DID Method
+    type: string
+    maxLength: 512
+  rubricEvaluation:
```

Comment:
I'm unsure if the rubric supports perma URLs yet, would prefer to add it only when its stable, and we can add a bunch of entries at the same time.

I am concerned of link rot on this attribute.

Reply:
We allow links to arbitrary sites via the `specification` entry. I don't see what's that different with this?

Reply:
@msporny I am fine with the idea that it be a link to a WG note that points to evaluations... I am saying I have not seen an example of that yet, and so I fear that adding this field may be premature, and or that link structure might change as the rubric changes... I guess really I want to see a couple examples to wrap my head around how this field will be used.

Can we see links for did:key, did:web, did:ion? for example?

Reply:
Let's see what examples @jandrieu has ready to go. I agree that if we don't have at least one or two complete examples that it's premature to add this field.

Reply:
My though is that the registrant would like a URL to a stable document that is either: a) a single rubric-based evaluation (if there was only one), OR b) an index of pointers to multiple rubric-based evaluations. However see also my suggestion above about just using one "Informational Document" field.